tests/apps/actorinvocationapp/service.yaml

```diff
@@ -0,0 +1,46 @@
+# In e2e test, this will not be used to deploy the app to test cluster.
```

Comment:
What is this for? Do devs need this in their local environment?

Reply:
This was just following the standard of all the other e2e tests which provide a template service.yaml in case you want to interact with your app directly instead of just running the tests.